CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Next Release
 
 * Your contribution here.
+* [#264](https://github.com/intridea/hashie/pull/264): Questioning abc? returns a boolean value instead of key value in Hashie::Extensions::MethodReader included classes - [@Zloy](https://github.com/Zloy).
```

Comment:
This phrase sounds a bit backwards and "questioning?" is very English ;) Also quote `abc?`. How about just something like this: "Methods such as `abc?` return true/false with Hashie::Extensions::MethodReader".